altos-rust/port/cortex-m0/src/peripheral/usart/baudr.rs

```diff
@@ -0,0 +1,71 @@
+// Daniel Seitz and RJ Russell
+
+use super::super::Register;
+use super::defs::*;
+
+#[derive(Copy, Clone)]
+pub struct UsartBRR {
```

Comment:
Don't really need a wrapper struct since it's only containing one register. The other registers were put in wrappers since they were multiple registers with similar purposes.